core_types/src/cas/storage.rs

```diff
@@ -231,6 +235,180 @@ where
     }
 }
 
+pub struct EAVTestSuite;
+
+impl EAVTestSuite {
+    pub fn test_round_trip_test(
```

Comment:
`test_rount_trip_test` sounds like we are testing the round_trip_test.. ;)

Reply:
@lucksus yeah so i've been informally following a convention where `test_*` means "generate a test example thing" and `*_test` "a test for a thing"